backend/types/cere_custom_types.json

```diff
@@ -20,6 +20,7 @@
     "id": "TokenId",
     "metadata": "Vec<u8>"
   },
-  "Address": "IndicesLookupSource",
-  "LookupSource": "IndicesLookupSource"
+  "Address": "MultiAddress",
+  "LookupSource": "MultiAddress",
+  "AccountInfo": "AccountInfoWithDualRefCount"
```

Comment:
Why do we need this?

Reply:
I added `AccountInfoWithDualRefCount` to provide backward compatibility with the old Cere node. Without that field application returns "invalid address info" error in the console for the old node when integrated with a new polkadot.js version.